web/src/components/CitySelector.tsx

```diff
@@ -52,19 +56,28 @@ const CitySelector = ({ cities, language, filterText, stickyTop = 0 }: PropsType
     const safeCities = cities.filter(city => city.code !== '')
     const sorted = sort(safeCities)
     const groups = groupBy(sorted, city => city.sortCategory)
-    return transform(
-      groups,
-      (result: Array<ReactNode>, cities, key) => {
-        result.push(
-          <div key={key}>
-            <CityListParent stickyTop={stickyTop}>{key}</CityListParent>
-            {cities.map(city => (
-              <CityEntry key={city.code} city={city} language={language} filterText={filterText} />
-            ))}
-          </div>
-        )
-      },
-      []
+    const failureProps = {
+      goToPath: null,
```

Comment:
No need to pass `null` here, just don't pass anything at all.

Reply:
This will result in "undefined", which does not work with the Failure Component, it will set the path to "/" :
Failure = ({ errorMessage, goToPath = '/', ....

Reply:
Ahh you're right. Then I'd remove the default value and set it explicitly wherever that was used.